src/core_plugins/kibana/public/dashboard/dashboard_app.js

```diff
@@ -329,11 +330,13 @@ app.directive('dashboardApp', function ($injector) {
       function setDarkTheme() {
         chrome.removeApplicationClass(['theme-light']);
         chrome.addApplicationClass('theme-dark');
```

Comment:
I'm guessing this is still needed because dashboard is a combination of non-eui elements and eui elements, so for the time being we keep both dark theme applications around?

Reply:
Correct.